scripts/extensions/topbar-publish-date/src/publish-date.tsx

```diff
@@ -0,0 +1,25 @@
+import React = require('react');
+import {IArticle, ISuperdesk} from 'superdesk-api';
+
+export function getPublishDate(superdesk: ISuperdesk) {
```

Comment:
When an extension is registered, the `superdesk` API is populated on the window object. Which means you don't need this wrapper function, you can simply add this at the top of the file `import {superdesk} from './superdesk'`